.gitignore

```diff
@@ -4,3 +4,4 @@ zammad/charts
 zammad/Chart.lock
 zammad/index.yaml
 zammad/requirements.lock
+values.yaml
```

Comment:
this seems wrong :)

Reply:
Why should this be wrong? It's for a top-level custom values.yaml file that the user might locally place for their customizations. It's not for the commited `zammad/values.yaml`. That seems correct to me.